front/lib/email.ts

```diff
@@ -157,3 +157,30 @@ export async function sendAdminSubscriptionPaymentFailedEmail(
   };
   return sendEmail(email, message);
 }
+
+export async function sendAdminDataDeletionEmail({
```

Comment:
Copy: would warn for last email (flag last email or condition on remaining days, pref for 1)
Prefix title with "Last reminder: "
And last sentence "This is our last message before data deletion"

Reply:
(reason: personally if I get a message 3 days before without it telling me it's the last, I dismiss it thinking "I'll wait for the last message")

Reply:
That's a great point !